lib/rspec/support/differ.rb

```diff
@@ -182,9 +183,9 @@ def object_to_string(object)
             pp_value = PP.singleline_pp(object[key], "")
 
             "#{pp_key} => #{pp_value},"
-          end.join("\n")
+          end.join(LINE_BREAK)
         when String
-          object =~ /\n/ ? object : object.inspect
+          object =~ /#{LINE_BREAK}/o ? object : object.inspect
```

Comment:
Same here, leaving the primitive seems better than interpolating the constant each time.